### PR TITLE
set up project; fill out readme; add text to email converter button

### PR DESCRIPTION
**Pull request by sarahwelzgeselowitz**
This is a bit of a hack that I don't expect to need much maintenance (hence no unit tests etc.) but time will tell. Wanna make sure it's at least well enough documented that someone could figure it out in the future.

See README for project goals

All auto-generated files except the readme and `assets/iframe.html`

### CODE REVIEW

File: assets/iframe.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+              })
+        })
+
+    }
+
+    // Let's grab the ticket
+    client.get('ticket').then(function(data) {
+      const {ticket} = data;
+      if(ticket.via.channel=='sms'){
+            // Vanilla JS and not Jquqery  b/c this is super lightweight.
```
Comment (capnfabs) [inline assets/iframe.html]: Makes lots of sense 👍

File: translations/en.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "app": {
+    "name": "Zen Tunes",
```
Comment (capnfabs) [inline translations/en.json]: Do you want to fix these?

File: assets/iframe.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
+<html>
```
Comment (capnfabs) [inline assets/iframe.html]: Does this iframe get reloaded automatically on every ticket?

Comment (sarahwelzgeselowitz) [reply assets/iframe.html]: Yup! :)

**Pull request merged**